inst/learnR-chapters/chapter_55_to_60_learning.Rmd

```diff
@@ -266,14 +258,13 @@ This summary provides the estimates (coefficients), standard errors, and p-value
 
 Let's interpret these results:
 
-* The coefficient for wt tells us about the direction and strength of the relationship between weight and fuel efficiency. If the coefficient is positive, the relationship is positive, and if it's negative, the relationship is negative.
-
-* The standard error informs us about the uncertainty of the coefficient estimate. The smaller the standard error, the more confident we can be about the estimate.
+-   The coefficient for wt tells us about the direction and strength of the relationship between weight and fuel efficiency. If the coefficient is positive, the relationship is positive, and if it's negative, the relationship is negative.
```

Comment:
Looks like you rearranged this section which I'm totally not opposed to. What was the idea behind it? Thanks for doing this!

Reply:
it was put on there twice so I deleted one of them